packages/features/ee/teams/components/CreateANewTeamForm.tsx

```diff
@@ -135,12 +135,28 @@ export const CreateANewTeamForm = () => {
               <>
                 <Label>{t("team_logo")}</Label>
                 <div className="flex items-center">
-                  <Avatar
-                    alt=""
-                    imageSrc={value}
-                    fallback={<Plus className="text-subtle h-6 w-6" />}
-                    size="lg"
-                  />
+                  <label htmlFor="avatar-upload" className="cursor-pointer">
+                    <Avatar
+                      alt=""
+                      imageSrc={value}
+                      fallback={<Plus className="text-subtle h-6 w-6" />}
+                      size="lg"
+                    />
+                    <input
+                      type="file"
+                      id="avatar-upload"
+                      style={{ display: "none" }}
+                      onChange={(event) => {
+                        const inputElement = event.target as HTMLInputElement;
+                        const selectedFile = inputElement.files?.[0];
+                        if (selectedFile) {
+                          const newAvatar = URL.createObjectURL(selectedFile);
+                          newTeamFormMethods.setValue("logo", newAvatar);
+                          createTeamMutation.reset();
+                        }
+                      }}
+                    />
```

Comment:
@riteshsinghh-coder thanks for working on it but i don't think we need this improvement we already have imageuploader component which handles very well. 

It also make sure we don't cross size limit, only accepted files are images and handle cropping which this doesn't